packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -341,7 +339,7 @@ export function TransferPanel() {
 
   const tokenAllowanceApproval = async () => {
     setIsCctp(false)
```

Comment:
The `isCctp` state is the boolean the `TokenApprovalDialog` relies on
`useIsCctpTransfer` can only determine if the selected token is eligible for CCTP transfers

Not using `isCctp` and using the hook directly caused the contract address to be wrong (showing for CCTP when it's native bridge USDC deposit)

<img width="769" alt="image" src="https://github.com/user-attachments/assets/cfb3a035-f29d-4ecb-b431-b1c168c44836" />